tests/functional/spec/multiple-answers.spec.js

```diff
@@ -0,0 +1,38 @@
+import chai from 'chai'
+import {getRandomString, startQuestionnaire} from '../helpers'
+import personalDetailsPage from '../pages/surveys/answers/personal-details.page'
+import multipleAnswerSummaryPage from '../pages/surveys/answers/multiple-answer-summary.page'
+
+const expect = chai.expect
+
+describe('Error messages', function() {
+
+  it('Given a survey has multiple answers for a question when I save the survey then the summary shows all the answers', function() {
+    // Given
+    startQuestionnaire('multiple_answers.json')
+
+    // When
+    personalDetailsPage.setFirstName('Han')
+      .setSurname('Solo')
+      .submit()
+
+    // Then
+    expect(multipleAnswerSummaryPage.getFirstName()).to.equal('Han')
+    expect(multipleAnswerSummaryPage.getSurname()).to.equal('Solo')
+  })
+
+  it('Given I complete a survey has multiple answers for a question when I edit an answer then I appear on the page to edit my answer', function() {
```

Comment:
This doesn't read very well. I think it should either be 'Given I complete a survey **that** has...` or just drop the first bit and have 'Given a survey has multiple answers...'